RELEASE-NOTES.txt

```diff
@@ -1,3 +1,12 @@
+1.15.0
+------
+* Fix issue when multiple media selection adds only one image or video block on Android
+* Fix issue when force Touch app shortcut doesn't work properly selecting "New Photo Post" on iOS
+* Add Link Target (Open in new tab) to Image Block.
+* [iOS] DarkMode improvements.
+* [iOS] Update to iOS 11 and Swift 5
```

Comment:
I'm not sure about `[iOS] Update to iOS 11 and Swift 5`, since it's not an `user facing change`.

Reply:
Some context on why I added it: since we also add some notes to the GitHub Release of the library itself, I figured that such upgrade is important to mention there. And indeed, it's not user facing so, happy it didn't reach the release-notes in the main app.

Maybe we can starting thinking of a way to denote this kind of "news" that are important/non-trivial for the library but not useful to bubble up to the main apps.

Reply:
That's a good idea. If you look at [core releases](https://wordpress.org/news/2019/05/jaco/), there's often the news for users, and then a section of changes for developers.